### PR TITLE
add OrangePi R1 Plus LTS support

### PR DESCRIPTION
**Pull request by mingxiaoyu**
Hardware
--------
RockChip RK3328 ARM64 (4 cores)
1GB DDR3 RAM
2x 1000 Base-T
3 LEDs (LAN / WAN / SYS)
1 Button (Reset)
Micro-SD slot
USB 2.0 Port

Installation
------------
Uncompress the OpenWrt sysupgrade and write it to a micro SD card using dd.

### CODE REVIEW

File: target/linux/rockchip/patches-5.10/206-rockchip-rk3328-Add-support-for-OrangePi-R1-Plus-LTS.patch [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
++// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
++#include "rk3328-orangepi-r1-plus.dts"
++
++/ {
++	model = "Xunlong Orange Pi R1 Plus LTS";
++	compatible = "xunlong,orangepi-r1-plus-lts", "rockchip,rk3328";
++};
++
++/delete-node/ &rtl8211e;
++&dmc_opp_table {
```
Comment (AmadeusGhost) [inline target/linux/rockchip/patches-5.10/206-rockchip-rk3328-Add-support-for-OrangePi-R1-Plus-LTS.patch:39]: Why disabled?

File: target/linux/rockchip/patches-5.10/206-rockchip-rk3328-Add-support-for-OrangePi-R1-Plus-LTS.patch [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
++
++        mdio {
++                compatible = "snps,dwmac-mdio";
++                #address-cells = <1>;
++                #size-cells = <0>;
++
++                ethphy3: ethernet-phy@0 {
++                        reg = <0x0>;
++                        keep-clkout-on;
++                       reset-gpios = <&gpio1 RK_PC2 GPIO_ACTIVE_LOW>;
```
Comment (AmadeusGhost) [inline target/linux/rockchip/patches-5.10/206-rockchip-rk3328-Add-support-for-OrangePi-R1-Plus-LTS.patch:69]: Indentation error.

File: target/linux/rockchip/patches-5.10/206-rockchip-rk3328-Add-support-for-OrangePi-R1-Plus-LTS.patch (hunk 1 above)
Comment (mingxiaoyu) [reply target/linux/rockchip/patches-5.10/206-rockchip-rk3328-Add-support-for-OrangePi-R1-Plus-LTS.patch:39]: just copy from orange official

Comment (AmadeusGhost) [reply target/linux/rockchip/patches-5.10/206-rockchip-rk3328-Add-support-for-OrangePi-R1-Plus-LTS.patch:39]: Please delete this and retest.

Comment (mingxiaoyu) [reply target/linux/rockchip/patches-5.10/206-rockchip-rk3328-Add-support-for-OrangePi-R1-Plus-LTS.patch:39]: sure.
can you tell me which line of code?  
38 ?

Comment (AmadeusGhost) [reply target/linux/rockchip/patches-5.10/206-rockchip-rk3328-Add-support-for-OrangePi-R1-Plus-LTS.patch:39]: Remove whole `dmc_opp_table` node.

Comment (mingxiaoyu) [reply target/linux/rockchip/patches-5.10/206-rockchip-rk3328-Add-support-for-OrangePi-R1-Plus-LTS.patch:39]: It seems that we should keep the code. 
Removing that ,LAN AND WAN  are not work.

**Comment by 1715173329** [comment]
IMHO, this is totally __UNACCEPTABLE__.

The most unacceptable part is patch for yt8531c, which is crappy, ugly and breaks yt8521s (coolsnowwolf/lede#8719).

**Comment by mingxiaoyu** [comment]
> IMHO, this is totally **UNACCEPTABLE**.
> 
> The most unacceptable part is patch for yt8531c, which is crappy, ugly and breaks yt8521s ([coolsnowwolf/lede#8719](https://github.com/coolsnowwolf/lede/issues/8719)).

**Please compare code !!!!!!!!!!!!!!!!!!!!!!!**
The guy copy whole code to LEDE.
I only add the yt8531c to patches.

@1715173329

**Comment by mingxiaoyu** [comment]
IMG_20220124_151044_364
it is not   break  R2C ！！！！！！！！！！！！！！！！！！！！！！！！！

**Pull request closed without merging**